.github/workflows/self-push.yml

```diff
@@ -6,12 +6,10 @@ on:
       - push-ci
       - ci_*
       - ci-*
-    paths:
-      - "src/**"
-      - "tests/**"
-      - ".github/**"
-      - "templates/**"
-      - "utils/**"
+  workflow_run:
+    workflows: ["Check for dependency modification"]
```

Comment:
Just a remark (probably redundant now as this block is going to be removed): this should not be `Check for dependency modification` as it doesn't exist in `transformers`.